src/platform/linker/cl_macros.h

```diff
@@ -35,7 +35,7 @@
 
 #define UNDEF_SYMB_TYPE 0x1
 #define EXPORTED_SYMB_TYPE 0x2
-#define MAX_SYMBOLS 1024
+#define MAX_SYMBOLS 1024*64
```

Comment:
I think it's probably good style to put parens around this like:
`#define MAX_SYMBOLS (1024 * 64)`
I don't think it really matters here, but it's better to be obviously hygienic.